config/plugins/visualizations/h5web/static/script.css

```diff
@@ -1,2 +0,0 @@
ckground-color:var(--secondary-bg);flex:none;justify-content:space-between;padding:0 .5rem;font-size:.875rem;display:flex}._breadCrumbs_csycc_11{min-width:0;font-size:inherit;font-weight:inherit;white-space:nowrap;flex:1;align-items:center;margin:0 .5rem 0 0;line-height:1.3;display:flex;overflow:hidden}._crumbButton_csycc_24{min-width:0;align-items:center;display:flex;position:relative}._crumbButton_csycc_24[data-current]{flex:none;padding-right:1rem;font-weight:600}._crumbButton_csycc_24:hover{text-decoration:underline}._crumb_csycc_24{text-overflow:ellipsis;margin:0 .125rem .125rem;overflow:hidden}._separator_csycc_48{min-width:.75rem;flex:none}._copyIcon_csycc_53{margin:2px 0 1px;font-size:.9rem;display:none;position:absolute;top:calc(50% - .125rem);right:.875rem;transform:translate(100%,-50%)}._crumbButton_csycc_24:hover>._copyIcon_csycc_53{display:flex}._table_z9o09_1{width:100%;text-align:left;border-collapse:collapse;margin-bottom:.25rem}._table_z9o09_1:not(:first-child){margin-top:1rem}._table_z9o09_1 th{font-weight:500}._table_z9o09_1 thead th{height:var(--toolbar-height);background-color:var(--primary-bg);padding:0 1rem;font-size:1.25rem}._table_z9o09_1 tbody th,._table_z9o09_1 tbody td{vertical-align:top;padding:.5rem 1rem}._table_z9o09_1 tbody th{width:15%}._table_z9o09_1 tbody td._raw_z9o09_33{padding:0}._table_z9o09_1 tbody tr:first-child>th,._table_z9o09_1 tbody tr:first-child>td{padding-top:1rem}._table_z9o09_1 tbody tr:first-child>td._raw_z9o09_33{padding-top:.5rem}._metadataViewer_z9o09_46{scrollbar-width:thin;overflow-y:scroll}._empty_z9o09_51{padding:1rem}._attrValue_z9o09_55{align-items:center;display:flex}._attrValueBtn_z9o09_60{color:var(--secondary-darker);align-items:center;font-weight:600;display:flex}._attrValueBtn_z9o09_60:hover{text-decoration:underline}._attrValueBtn_z9o09_60>svg{margin-left:.25rem;transform:translateY(1px)}._attrValueWrapper_z9o09_77{margin:0 -.25rem;display:flex}._attrValueWrapper_z9o09_77>span{margin:0 .25rem}._heading_140is_1{cursor:pointer;padding:.5rem 1rem}._heading_140is_1:hover{text-decoration:underline}._content_140is_10{font-family:var(--monospace);margin:0;padding:.75rem 1rem}._selector_1d1fc_1{font-size:.875em;display:flex}._btn_1d1fc_6{outline-offset:-1px;border-bottom:.25rem solid #0000;align-items:center;padding:.125rem 1rem 0;line-height:1;transition:border-color ease-out;display:flex}._btn_1d1fc_6:hover,._btn_1d1fc_6[aria-selected=true]{transition-duration:.2s,.2s}._btn_1d1fc_6:hover{border-bottom-color:var(--secondary-light)}._btn_1d1fc_6[aria-selected=true]{border-bottom-color:var(--secondary);font-weight:600}._icon_1d1fc_31{width:1.125em;height:1.125em;margin-bottom:-.0625em;margin-right:.5rem}._visManager_ghsny_1{min-height:0;scrollbar-width:thin;flex-direction:column;flex:1;display:flex}._visBar_ghsny_9{height:var(--toolbar-height);background-color:var(--secondary-light-bg);flex:none;justify-content:space-between;display:flex}._visArea_ghsny_17{min-height:0;flex:1;display:flex}
```

Comment:
Just double-checking: this file is not rebuilt for me when running `./scripts/common_startup.sh`, but it may be due to the errors reported at https://matrix.to/#/!XciPHtOkuKNqKoVeKD:gitter.im/$puPFPKnkiJsOfbchZfzQPM_6kI1v4yBAZQg7VKOmpp4?via=gitter.im&via=matrix.org by @davelopez .

Reply:
This is correct.  I'll have a PR for that in a sec.